<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Snowfall</class>
 <widget class="QWidget" name="Snowfall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(92, 167, 242);</string>
  </property>
  <widget class="QLineEdit" name="name_inf">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>Имя получателя(не больше 15 символов)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snowfall_color">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>160</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>Цвет снежинки</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="back_color">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>110</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>Цвет фона</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_button3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>15</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введи в каждую строку выбранный цвет из перечисленных (цвет
должен быть написан в той же форме, что представлена ниже
Выбор цвета: синий, жёлтый, зелёный, чёрный, красный,
коричневый, розовый, голубый, серый, белый</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
